modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -107,16 +102,7 @@ because it guarantees that cached objects are available for as long as they are
 The Redis PHP module must be at least version 2.2.6 or higher.
 If you are running a Linux distribution that does not package the supported versions of this module — or does not package Redis at all — see xref:installing-redis-on-other-distributions[Installing Redis on other distributions].
 
-TIP: Debian Jessie users, please see this https://github.com/owncloud/core/issues/20675#issuecomment-159202901[GitHub discussion] if you have problems with LDAP authentication when using Redis.
```

Comment:
Would it not make sense to keep this line?

Reply:
Not if we agreed to make the docs ubuntu only.

Additionally:

> Debian “jessie” Release Information
> Debian 8.11 was released June 23rd, 2018. Debian 8.0 was initially released on April 26th, 2015. The release included many major changes, described in our press release and the Release Notes.
> 
> Debian 8 has been superseded by Debian 9 (stretch). Regular security support updates have been discontinued as of June 17th, 2018.
> 
> Jessie also benefits from Long Term Support (LTS) until the end of June 2020.

Reply:
I thing that it should be differenciated between describing commands based on Ubuntu vs giving helpful hints for other distros where not the command is in forground but the tip.

Reply:
> Not if we agreed to make the docs ubuntu only.
> 
> Additionally:
> 
> > Debian “jessie” Release Information
> > Debian 8.11 was released June 23rd, 2018. Debian 8.0 was initially released on April 26th, 2015. The release included many major changes, described in our press release and the Release Notes.
> > Debian 8 has been superseded by Debian 9 (stretch). Regular security support updates have been discontinued as of June 17th, 2018.
> > Jessie also benefits from Long Term Support (LTS) until the end of June 2020.

Fair enough.